firefox-ios/Client/Frontend/Microsurvey/Prompt/MicrosurveyPromptView.swift

```diff
@@ -47,12 +48,23 @@ class MicrosurveyPromptView: UIView, ThemeApplicable, Notifiable {
     private var logoWidthConstraint: NSLayoutConstraint?
     private var logoHeightConstraint: NSLayoutConstraint?
 
+    private var logoSize: CGSize {
+        let contentSizeCategory = UIApplication.shared.preferredContentSizeCategory
+        return contentSizeCategory.isAccessibilityCategory ? UX.logoLargeSize : UX.logoSize
```

Comment:
Is there a reason we use two different sizes for icons instead of scaling the icon using `UIFontMetrics.default.scaledValue(for: )`?

Reply:
thanks for calling this out @thatswinnie! Let me confirm with design on this and see if any issues, in the meantime I updated the code (can always revert).

Reply:
@thatswinnie design says this is good to scale like this, thank you for pointing this out. Will keep in mind to discuss using this when scaling images for a11y.